src/model/docker.ts

```diff
@@ -36,12 +37,13 @@ class Docker {
     --rm \
     ${ImageEnvironmentFactory.getEnvVarString(parameters)} \
     ${baseOsSpecificArguments} \
-    ${image}`;
+    ${image} -- \
```

Comment:
I would keep this on two lines, or put the two hyphens on the line that passed the actual command
```suggestion
    ${image} \
    -- \
```

Reply:
(already committed)